src/styles/components/index.scss

```diff
@@ -1,4 +1,4 @@
-@import 'src/styles/globals.scss';
+@import "src/styles/globals";
```

Comment:
I'm just noticing this file for the first time. It doesn't look like it gets imported anywhere, and it doesn't really follow the ITCSS pattern that we're going for. I would delete this file, leaving `components/_header.scss` and `globals.scss` as our only SCSS files for now.